source/modulo_components/include/modulo_components/ComponentInterface.h

```diff
@@ -1,8 +1,20 @@
 #pragma once
 
 #include <state_representation/parameters/ParameterMap.hpp>
-#include <rclcpp/node.hpp>
-#include <rclcpp_lifecycle/lifecycle_node.hpp>
+#include <state_representation/space/cartesian/CartesianPose.hpp>
+#include <rclcpp/parameter.hpp>
+#include <rclcpp/create_timer.hpp>
+
+#include <tf2_ros/buffer.h>
+#include <tf2_ros/transform_listener.h>
+#include <tf2_ros/transform_broadcaster.h>
+
+#include <modulo_new_core/translators/ReadStateConversion.hpp>
+#include <modulo_new_core/translators/WriteStateConversion.hpp>
+#include <exceptions/PredicateNotFoundException.hpp>
```

Comment:
this file should be moved and the include updated accordingly
```suggestion
#include "modulo_components/exceptions/PredicateNotFoundException.hpp"
```